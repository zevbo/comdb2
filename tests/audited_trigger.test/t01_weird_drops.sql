CREATE TABLE t1(a int, b int)$$

CREATE LUA TRIGGER t1_audit ON (table t1 FOR INSERT AND UPDATE AND DELETE) AUDITED

CREATE LUA TRIGGER t1_audit ON (table t1 FOR INSERT AND UPDATE AND DELETE) AUDITED

DROP LUA TRIGGER t1_audit

CREATE LUA TRIGGER t1_audit ON (table t1 FOR INSERT AND UPDATE AND DELETE) AUDITED

DROP TABLE '$audit_t1'

CREATE LUA TRIGGER shoutout_to_alexw ON (table t1 FOR INSERT AND UPDATE AND DELETE) AUDITED

SELECT * FROM comdb2_tables

DROP LUA TRIGGER t1_audit
DROP LUA TRIGGER shoutout_to_alexw 
DROP TABLE '$audit_t1'
DROP TABLE '$audit_t1$2'
DROP TABLE t1