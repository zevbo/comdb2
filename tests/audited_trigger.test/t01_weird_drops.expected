[CREATE TABLE t1(a int, b int)] rc 0
[CREATE LUA AUDIT TRIGGER t1_audit ON (table t1 FOR INSERT AND UPDATE AND DELETE)] rc 0
[DROP LUA TRIGGER t1_audit] rc 0
[CREATE LUA AUDIT TRIGGER t1_audit ON (table t1 FOR INSERT AND UPDATE AND DELETE)] rc 0
[DROP TABLE '$audit_t1'] rc 0
[CREATE LUA AUDIT TRIGGER shoutout_to_alexw ON (table t1 FOR INSERT AND UPDATE AND DELETE)] rc 0
(tablename='$audit_t1')
(tablename='$audit_t1$2')
(tablename='sqlite_stat1')
(tablename='sqlite_stat4')
(tablename='t1')
[SELECT * FROM comdb2_tables ORDER BY tablename] rc 0
[DROP LUA TRIGGER t1_audit] rc 0
[DROP LUA TRIGGER shoutout_to_alexw] rc 0
[DROP TABLE '$audit_t1'] rc 0
[DROP TABLE '$audit_t1$2'] rc 0
[DROP TABLE t1] rc 0
